docs/learn/encyclopedia/data-format/xdr.mdx

```diff
@@ -16,6 +16,27 @@ Since XDR is a binary format and not as widely known as simpler formats like JSO
 
 In addition, the Horizon API server generally exposes the most important parts of the XDR data in JSON, so they are easier to parse if you are not using an SDK. The XDR data is still included (encoded as a base64 string) inside the JSON in case you need direct access to it.
 
+
+## JSON and XDR Conversion Schema
+
+:::info
+
+SDF has published libraries which support the canonical and round-trippable conversion from XDR to JSON, and from JSON to XDR, without loss of information.
+
+:::
+
```

Comment:
The info block here can probably be dropped. SDF publishes a lot of things so calling that out isn't particularly impactful.
```suggestion
```